src/entity/ExperimentVariant.ts

```diff
@@ -13,4 +17,7 @@ export class ExperimentVariant {
 
   @Column({ type: 'text', default: null })
   value: string;
+
+  @Column({ type: 'text' })
+  type: ExperimentVariantType;
```

Comment:
Just added this for future proofing, if we start storing other experiment values in the table, we can store them by type.